.cirrus.yml

```diff
@@ -33,8 +33,8 @@ debian_version_task:
 
     container:
         matrix:
-          - image: debian:9.13
-          - image: debian:10.8
+          - image: debian:buster
+          - image: debian:bullseye
```

Comment:
Usually we try to pin the versions to avoid false positives errors on CI. This will make CI update more frequently and can cause problems. We can revisit that, and discuss this, of course.

Reply:
I was thinking this kind of comment was a possible answer to this PR while making it. I still went ahead and used the codename because I consider Debian to be extremely conservative in updates for point releases and I'm not expecting a lot of problems.
That said, I'm ok to continue pinning to the versions exactly for the same reasons: point releases are very conservative and even if we don't update often, it won't change a lot the result of the tests.

Reply:
I see your point, it is just a matter of deciding what we should follow. Even with a conservative distro, like debian, buster had 9 updates, since mid-2019. I'm ok with that approach too, I just would like to make sure that we are agreeing on this change and giving more visibility to this change.

So, let see with others has to say. @clebergnu, thoughts ?

Reply:
FWIW, I did a force-push using version instead of codename (except for bullseye, that'll be a timestamp until is released)